packages/engine/Specs/Scene/BillboardCollectionSpec.js

```diff
@@ -28,1845 +28,1879 @@ import {
 import createScene from "../../../../Specs/createScene.js";
 import pollToPromise from "../../../../Specs/pollToPromise.js";
 
-describe(
```

Comment:
Just curious why so many changes to this spec? Are these related to the texture atlas refactor? I cannot tell from a quick first glance

Reply:
A lot of this is whitespace changes. I turned a lot of promise chains to async/await in the process of updating them.